CONTRIBUTING.md

```diff
@@ -1,80 +1,29 @@
-# WIP: Internal contributing guide
+# Contributing
 
-## Weave Gitops :heart:s your contributions
+Weave Gitops is a [Mozilla Public License(MPL2)](LICENSE) project. This is a Weaveworks
+open source product with a community led by Weaveworks :heart:
 
-Thank you for taking the time to contribute to Weave Gitops!
+We welcome improvements to reporting issues and documentation as well as to code.
 
-This guide **is a work in progress** but aims to cover all aspects of how to
-interact with the project and how to get involved in development as smoothly as possible.
-
-If we have missed anything you think should be included, or if anything is not
-clear, we also accept contributions to this contribution doc :smile:.
-
-
-## Building the binaries
-To build the `gitops` binary locally you can run `make gitops`. This will create a `gitops`
-binary in the `bin/` directory. Similarly running `make gitops-server` will create a
-`gitops-server` binary in the same directory.
-
-
-## Testing
-We have 3 layers of testing:
-- Unit testing
-- Integration testing
-- Acceptance testing
-
-We aim to follow the [testing pyramids best practises](https://martinfowler.com/articles/practical-test-pyramid.html)
-with the focus on a large number of units tests, fewer integration tests and even fewer acceptance tests.
+## Understanding how to run development process
 
+The [internal guide](doc/development-process.md) **is a work in progress** but aims to cover all aspects of how to
```

Comment:
internal guide could imply that it is weaveworks-only, how about development guide?

Reply:
it is mostly internal, for now.